yamlh.go

```diff
@@ -459,31 +459,28 @@ type yamlSimpleKey struct {
 type yamlParserState int
 
 const (
-	yaml_PARSE_STREAM_START_STATE yamlParserState = iota
```

Comment:
Is this diff necessary? afaict you added a single comment that forced reformatting of the rest.
Can you introduce that comment (if really needed) in a way that doesn't make this big diff?